service/controller/v11/key/key.go

```diff
@@ -40,7 +40,7 @@ const (
 	CoreosVersion        = "1688.5.3"
 
 	K8SEndpointUpdaterDocker  = "quay.io/giantswarm/k8s-endpoint-updater:df982fc73b71e60fc70a7444c068b52441ddb30e"
-	K8SKVMDockerImage         = "quay.io/giantswarm/k8s-kvm:4438d70d2181af66ea2b90c7f3bc74d1aa3c55a1"
+	K8SKVMDockerImage         = "quay.io/giantswarm/k8s-kvm:03f5254c5dc7bb9c8ad6c881136e9a96771d504c"
```

Comment:
See https://github.com/giantswarm/k8s-kvm/pull/19. For now only introducing a dumb sleep. I go after this in the next step and make it better. I want to get this here out now because it works well enough for us. I want to improve the process to be more event based though. Stay tuned folks.